server/schedulers/scatter_range.go

```diff
@@ -193,6 +193,15 @@ func (l *scatterRangeScheduler) EncodeConfig() ([]byte, error) {
 }
 
 func (l *scatterRangeScheduler) IsScheduleAllowed(cluster opt.Cluster) bool {
+	return l.allowBalanceLeader(cluster) || l.allowBalanceRegion(cluster)
+}
+
+func (l *scatterRangeScheduler) allowBalanceLeader(cluster opt.Cluster) bool {
+	// TODO: add metrics for leader limit
```

Comment:
Will it continue to be added to this PR?

Reply:
done